### ISSUE TITLE
Bugs - Lighthouse issues - Checking for accessibility

### ISSUE THREAD

**Issue by pkbullock** [pkbullock/m365-bookmark-generator#20]
- Accessibility, text does not meet the contract ratio
- Use of H4 in some tabs.
- SEO - page indexing and description

[bookmarks.pkbullock.com-20250908T191020.html](https://github.com/user-attachments/files/22216766/bookmarks.pkbullock.com-20250908T191020.html)
